server/unicast.go

```diff
@@ -22,9 +22,13 @@ func (h *Unicast) DefaultMethod(r *http.Request, args *UnicastArgs, reply *struc
 		if err != nil {
 			log.Printf("Call error: %v", err)
 		}
-		argumentsForCall := make([]interface{}, len(args.ServerArgs)+1)
+		argumentsForCall := make([]interface{}, 0, len(args.ServerArgs)+1)
```

Comment:
You don't need that at all. A zero-size will just be enough, just `append` it later.